packages/react/src/context/slash-id-context.tsx

```diff
@@ -15,6 +15,7 @@ export type StorageOption = "memory" | "localStorage";
 
 export interface SlashIDProviderProps {
   oid: string;
+  token?: string;
```

Comment:
please adjust the name to `initialToken` :)

Reply:
Done already!